client/app/telecom/telephony/service/assist/support/translations/Messages_fr_FR.xml

```diff
@@ -3,7 +3,7 @@
    <translation id="telephony_line_assist_support_title" qtlid="336207">Guides et support</translation>
 
    <translation id="telephony_line_assist_support_guides_title" qtlid="300370">Guides</translation>
-   <translation id="telephony_line_assist_support_guides_explain" qtlid="387796">Pour vous aider dans la configuration et l'utilisation de votre téléphonie, <a href="{{ url }}" target="_blank" title="Accéder à nos guides">nos guides sont à votre disposition<i class="ovh-font ovh-font-newtab left-space-m4" /></a>.</translation>
+   <translation id="telephony_line_assist_support_guides_explain" qtlid="387796">Pour vous aider dans la configuration et l'utilisation de votre téléphonie, <a href="{{ url }}" target="_blank" title="Accéder à nos guides">nos guides sont à votre disposition<i class="ovh-font ovh-font-newtab ml-4" /></a>.</translation>
```

Comment:
It's not yours but I think the `title` attribute from the `<a>` element should have his own key trad.

Reply:
IMHO I think it's over thinking this, in fact we should simply avoid putting html in translations